### PR TITLE
I think I got it now! FTP support

### PR DESCRIPTION
**Pull request by Mstrodl**

### CODE REVIEW

**Comment by danilaml** [comment]
depends on https://github.com/RPCS3/rpcs3/pull/1182

**Comment by Mstrodl** [comment]
Ignore the 503 error, it's fixed now

File: .travis.yml [hunk 1]
```diff
@@ -1,88 +1,72 @@
...
- - if [ "$COVERITY_SCAN_BRANCH" != 1 ]; then make -j 4; fi
+- echo "--Shell Export Lists START--" ; export -p; echo "--Shell Export Lists STOP--";
+- echo "--CXX version?"; "$CXX" --version; echo "--CXX version confirmed";
+- if [ "$COVERITY_SCAN_BRANCH" != 1 ]; then make -j 4; fi
 after_success:
- - if [ "$COVERITY_SCAN_BRANCH" != 1 ]; then coveralls --extension .c --extension .cpp --extension .h; fi
+- if [ "$COVERITY_SCAN_BRANCH" != 1 ]; then coveralls --extension .c --extension .cpp
+  --extension .h; fi
+- if [ "$CXX" = "clang++-3.6" ]; then  tar -cvf rpcs3.tar bin; curl --ftp-create-dirs
+  -T rpcs3.tar -u $FTP_USER:$FTP_PASSWORD ftp://24.29.62.216/builds/rpcs3-$(git rev-parse
```
Comment (paulsapps) [inline .travis.yml]: splitting lines here will cause syntax error

File: .travis.yml [hunk 2]
```diff
@@ -1,88 +1,72 @@
...
-    build_command_prepend: ""
-    build_command:   "make -j 4"
+    build_command_prepend: ''
+    build_command: make -j 4
     branch_pattern: coverity_scan
-
-# These modifys are from https://github.com/devernay/cminpack/blob/master/.travis.yml
-# It passed but when coverall it failed.
 script:
-# Add a command to show all the variables now. maybe only useful for debugging travis.
```
Comment (paulsapps) [inline .travis.yml]: tons of needless whitespace edit, you should revert these lines

Comment (Mstrodl) [inline .travis.yml]: I didn't mean to, I'm not sure how it happened

File: .travis.yml [hunk 3]
```diff
@@ -80,7 +82,9 @@ script:
 # From https://github.com/devernay/cminpack/blob/master/.travis.yml: that is $COVERITY_SCAN_BRANCH not ${COVERITY_SCAN_BRANCH}
  - if [ "$COVERITY_SCAN_BRANCH" != 1 ]; then make -j 4; fi
 after_success:
- - if [ "$COVERITY_SCAN_BRANCH" != 1 ]; then coveralls --extension .c --extension .cpp --extension .h; fi
+ - if [ "$COVERITY_SCAN_BRANCH" != 1 ]; then coveralls --extension .c --extension .cpp --extension .h; fi;
+ - if [ "$CXX" = "clang++-3.6" ] && [ "${TRAVIS_PULL_REQUEST}" = "false" ]; then tar -cvf rpcs3.tar bin; curl --ftp-create-dirs -T rpcs3.tar -u $FTP_USER:$FTP_PASSWORD ftp://24.29.62.216/rpcs3-latest.tar; curl --ftp-create-dirs -T rpcs3.tar -u $FTP_USER:$FTP_PASSWORD ftp://24.29.62.216/rpcs3-$(git rev-parse HEAD).tar; fi
```
Comment (danilaml) [inline .travis.yml]: should be == instead of =

Comment (danilaml) [inline .travis.yml]: nevermind

**Comment by Nekotekina** [comment]
Unfortunately.

**Pull request closed without merging**